src/basic_recipes/voronoiplot.jl

```diff
@@ -177,6 +177,7 @@ function plot!(p::Voronoiplot{<:Tuple{<:DelTri.VoronoiTessellation}})
         if color === automatic
             # generate some consistent distinguishable colors
             cs = [sum(DelTri.get_generator(vorn, i)) for i in DelTri.each_generator(vorn)]
+            reverse!(cs)
```

Comment:
I wanted to make it possible to move a point slightly and not have all the colors change so much, but this makes it hard to also want distinguishable colors. So, just to get better default behaviour, I'll keep `reverse!` here. Don't need to do anything better.